### ISSUE TITLE
[Bug] JSX and missing operand for operator

### ISSUE THREAD

**Issue by yhnavein** [quick-lint/quick-lint-js#654]
Hi,
Consider the following piece of JSX code:

```jsx
return (
  <Root>
     {showImage && <PlaceholderImage />}
  </Root>
);
```

`quick-lint-js` marks `&&` and ` />` as invalid with the following explanation:

> missing operand for operatorquick-lint-js[E0026](https://quick-lint-js.com/errors/#E0026)
![image](https://user-images.githubusercontent.com/516709/154464329-73d90017-4cc6-4076-a88e-5b5b8c2dbd1b.png)

For me this code looks just fine

I am using version 2.1.0 bundled in VSCode

**Comment by strager** [comment]
Thanks for the report! This does look like a bug in quick-lint-js.

**Comment by strager** [comment]
Fixed in commit f402e4966226d1f1d9824e35658d3b8600cc20ba. I'll publish a new version of the quick-lint-js VS Code extension soon.

**Comment by strager** [comment]
I shipped quick-lint-js version 2.2.0 with the fix. Thanks again for reporting.
